roles/ood_jupyter/templates/custom-jupyter-form.yml

```diff
@@ -37,8 +37,12 @@ attributes:
     widget: text_area
     label: Environment Setup
     value: |
-      # Load required modules
-      module load Anaconda3
+      # The latest version of Anaconda3 with jupyter is loaded by default.
```

Comment:
This text is not appearing in my form.  I suspect an issue with the comment prefix causing it to be ignored.

I have not taken any action other than re-applying the role.  Typically that is sufficient.

Reply:
i think it's because of the thing i told you yesterday. It caches what you successfully submited last time.

Reply:
Ah. that may make some sense. That cache is even referenced by link at the bottom of the form.  Deleted the json file and got the default behavior.